libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/DefaultNotifiableEventResolver.kt

```diff
@@ -316,6 +317,17 @@ class DefaultNotifiableEventResolver @Inject constructor(
             }
             .getOrNull()
     }
+
+    private suspend fun NotificationContent.MessageLike.RoomMessage.getImageMimetype(): String? {
```

Comment:
Couldn't this be reused in the previous function (`fetchImageIfPresent`)?

Reply:
Yes, maybe. Also I do not see a better way to avoid the code duplication here. I do not like using `Pair`.